lib/esc.js

```diff
@@ -397,7 +398,7 @@ ESC.prototype.stop = function() {
   var history = state.history;
   var speed = this.type === "bidirectional" ? this.neutral : 0;
 
-  this.write(this.pin, Fn.fscale(speed, 0, 100, 0, 180));
+  this.write(this.pin, Fn.fscale(speed, 0, 100, this.pwmRange[0], this.pwmRange[1]));
 
   history.push({
     timestamp: Date.now(),
```

Comment:
This is going to drastically improve the usefulness of `ESC` objects!

Reply:
Bonus! One of these days I'm going to have to learn more about ESC's

Reply:
I don't think the request for review will close unless I edit line 401, but this wasn't really a change request... Not sure what will happen if I click dismiss review.